apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/style-tour.scss

```diff
@@ -0,0 +1,135 @@
+@import '~@wordpress/base-styles/colors';
+@import '~@wordpress/base-styles/mixins';
+@import '~@wordpress/base-styles/variables';
+@import '~@wordpress/base-styles/z-index';
+
+.wpcom-editor-welcome-tour-frame {
+	background-color: $white;
+	border-radius: 2px;
+	bottom: 44px;
+	display: inline;
+	left: 16px;
+	position: fixed;
+}
+
+.welcome-tour-card__heading {
+	font-size: 1.125rem; /* stylelint-disable-line */
+	margin: 0.5rem 0;
+}
+
+.welcome-tour-card__description {
+	font-size: 0.875rem;
+	line-height: 1.5rem;
+	margin: 0;
+
+	.components-button {
+		height: auto;
+		line-height: 1;
+		text-decoration: underline;
+		padding: 0 0 0 4px;
+	}
+}
+
+.wpcom-editor-welcome-tour__resume-btn {
+	border-radius: 2px;
+	box-shadow: 0 2px 6px rgba( 60, 66, 87, 0.08 ), 0 0 0 1px rgba( 60, 66, 87, 0.16 ),
+		0 1px 1px rgba( 0, 0, 0, 0.08 );
+	background-color: $white;
+	color: $black;
+
+	&.components-button {
+		height: 44px;
+	}
+}
+
+.wpcom-editor-welcome-tour-card-frame {
+	position: relative;
+
+	// TODO update markup/css  if this isn't ok
```

Comment:
Is this still a TODO? I wasn't sure what might need to be changed in a follow-up?

Reply:
removed. :)